plugins/git-plugin/src/main/groovy/org/rundeck/plugin/scm/git/imp/actions/ImportJobs.groovy

```diff
@@ -134,6 +134,7 @@ class ImportJobs extends BaseAction implements GitImportAction {
                     jobsChanged.add(importResult.getJob())
                 }
                 plugin.importTracker.trackJobAtPath(importResult.job,walk.getPathString())
+                success = true
```

Comment:
This is weird, in the line 73 `success` is initialized as true, why this needs to be set as true again if nothing fails in between?

Reply:
In the line 84, success is set to false.